keras_cv/layers/preprocessing/random_hue.py

```diff
@@ -28,40 +28,45 @@ class RandomHue(tf.keras.__internal__.layers.BaseImageAugmentationLayer):
     hue channel (H) by delta. The image is then converted back to RGB.
 
     Args:
-        factor: Either a tuple of two floats or a single float. `factor` controls the
-            extent to which the image saturation is impacted. `factor` =
-            `0.0`, `0.5` or `1.0` makes this layer perform a no-op operation.
+        factor: A tuple of two floats, a single float or `keras_cv.FactorSampler`.
+            factor` controls the extent to which the image saturation is impacted.
+            `factor`=`0.0`, `0.5` or `1.0` makes this layer perform a no-op operation.
```

Comment:
I think this will need some update (since you mentioned in the PR description that the 0.0 is noop and 1.0 is the strongest augmentation.)

Reply:
Good point - updated.